src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1565,7 +1565,7 @@ internal enum ErrorCode
         ERR_NewBoundWithUnmanaged = 8375,
         ERR_UnmanagedConstraintMustBeFirst = 8376,
         ERR_UnmanagedConstraintNotSatisfied = 8377,
-        ERR_UnmanagedConstraintWithLocalFunctions = 8378,
```

Comment:
>ERR_UnmanagedConstraintWithLocalFunctions [](start = 8, length = 41)

I'm releasing this slot to other features, as this PR is still shipping with 15.7. Should we hold it indefinitely instead?